lib/redis/store/factory.rb

```diff
@@ -62,7 +62,7 @@ def self.normalize_key_names(options)
       end
 
       def self.host_options?(options)
-        if options.keys.any? {|n| [:host, :db, :port].include?(n) }
+        if options.keys.any? {|n| [:host, :db, :port, :path].include?(n) }
```

Comment:
this is a really weird method...I think it should be returning either `true` or `false`, therefore the next 4 lines shouldn't be necessary.

Reply:
heh, for sure. I'll clean it up. do you want me to squash my cleanups into the same commit, or are you okay with multiple commits in the PR?

Reply:
I prefer squashed commits.